webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/NetworkingInterfaces/NICDescriptionList.js

```diff
@@ -55,8 +56,16 @@ const NICDescriptionList = ({ status, foremanInterface }) => {
               {identifier || (
                 <span className="disabled-text">{__('No name')}</span>
               )}
-              {primary && <TagIcon title={__('Primary')} />}
-              {provision && <HddIcon title={__('Provision')} />}
+              {primary && (
+                <Icon style={{ marginLeft: '5px' }}>
+                  <TagIcon title={__('Primary')} />
+                </Icon>
+              )}
+              {provision && (
+                <Icon style={{ marginLeft: '5px' }}>
```

Comment:
Could you please move the style here into foreman/webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/NetworkingInterfaces/NetworkingInterfaces.scss
I find it confusing since there is margin left for the icons set there as well

Reply:
I think the current margin left only applied to the container itself, rather than to the icons within. Anyway I moved the style to the scss file.